ape/ape_prompts/reflexion.prompt

```diff
@@ -0,0 +1,58 @@
+---
+model: claude-3-5-sonnet-20240620
+description: Create a new prompt based base prompt and reflection
+inputs: 
+    question: The question to be answered
+    llm_output: The output of the LLM
+    llm_reasoning: The reasoning of the LLM
+    ground_truth_reasoning: The reasoning of the ground truth
+    ground_truth_answer: The answer of the ground truth
+outputs:
+    prompt: A new prompt based on the base prompt and reflection
```

Comment:
it should be `question` and `insight`?

and `response_format  = {type:json_object}`is needed